console/src/components/CompactHeader/CompactHeader.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import kind from '@enact/core/kind';
+import Button from '@enact/agate/Button';
+
+import css from './CompactHeader.less';
+import {getPanelIndexOf} from '../../App';
+
+const CompactHeader = kind({
+	name: 'CompactHeader',
+
+	propTypes: {
+		children: PropTypes.node,
+		isFor: PropTypes.string,
```

Comment:
I'm on the fence about the `isFor` prop name. It's readable -- `isFor="climate"` ~ "the header _is for_ the climate component" -- but it isn't a format we've used elsewhere. Are there other names you considered?

Reply:
no, can you suggest?

Reply:
I think the usage of `isFor` is essentially a unique id, or a key for each header. Some other options would be something like `id`, `name`  or even `index` as this is only used for the index. IMHO, I don't quite understand why we need the mapping of tab indices to the panel titles, but yeah...